### PR TITLE
ignore commented images (AST-65541)

### PR DESCRIPTION
**Pull request by davidkesoshvili**

### CODE REVIEW

File: internal/extractors/dockerfile_extractor.go [hunk 1]
```diff
@@ -59,6 +60,10 @@ func extractImagesFromDockerfile(logger *logger.Logger, filePath types.FilePath,
 		// Replace placeholders with values from mergedEnvVars and argsAndEnv
 		line = replacePlaceholders(line, mergedEnvVars, argsAndEnv)
 
+		if strings.HasPrefix(strings.TrimSpace(line), "#") {
```
Comment (greensd4) [inline internal/extractors/dockerfile_extractor.go:63]: comments always start like this?

File: internal/extractors/dockerfile_extractor.go [hunk 2]
```diff
@@ -59,6 +60,10 @@ func extractImagesFromDockerfile(logger *logger.Logger, filePath types.FilePath,
...
+			continue
```
Comment (greensd4) [inline internal/extractors/dockerfile_extractor.go:64]: pls add a test for this scenario

File: internal/extractors/dockerfile_extractor.go (hunk 1 above)
Comment (davidkesoshvili) [reply internal/extractors/dockerfile_extractor.go:63]: yeah

File: internal/extractors/dockerfile_extractor.go (hunk 2 above)
Comment (davidkesoshvili) [reply internal/extractors/dockerfile_extractor.go:64]: updated test

**Review by greensd4 — APPROVED** [review]

**Pull request merged**